### ISSUE TITLE
Replicating paper's results

### ISSUE THREAD

**Issue by bartmch** [arpanmangal/CovidAID#8]
Hi, I was just trying to replicate the ROC's you presented in your paper using the test_set you described in prepare_data.py. I'm executing trainer.py with your proposed settings:
```
                "--mode","test",
                "--checkpoint","path_to_CovidAID_transfered.pth.tar",
                "--cm_path","path_to_plots/cm_best",
                "--roc_path","path_to_plots/roc_best",
                "--combine_pneumonia"
```
I see that default batch size is 64 which brings the total inputs.shape to [640,3,224,224] (default 10 crops). This doesn't fit on GPU's memory. I was wondering what other settings you used to validate on the test set? E.g.: did you added a specific batch size "--bs","1" or did you use the distributed processing - if so can you explain to me how to use this in your code?
Thanks!

**Comment by arpanmangal** [comment]
`batch_size` at test time does not matter. You can use any size which fits in your GPU memory. For training, see sec. 3.3 of our paper for hyper-parameters.

**Comment by bartmch** [comment]
I just lowered the batch size and it runs fine (testing bs shouldn't matter - you're right).
A more urgent question: I created the ROCs and CMs for the data split in the code (which reflects the paper). When I want to replicate the results using:
`["--mode","test","--freeze","--checkpoint","path_to/CovidAID_transfered.pth.tar","--bs","8","--cm_path","path_to/plots/cm_best","--roc_path","path_to/plots/roc_best","--combine_pneumonia"]`
I get the following ROC and CM:
![image](https://user-images.githubusercontent.com/10415569/84258454-04746580-ab28-11ea-9d64-e7b979529d04.png)
![image](https://user-images.githubusercontent.com/10415569/84258471-0a6a4680-ab28-11ea-8b50-db7365169af8.png)
![image](https://user-images.githubusercontent.com/10415569/84258484-0fc79100-ab28-11ea-9ee3-b3c9a7b75878.png)

**Comment by arpanmangal** [comment]
Which pre-trained model did you use? Did you use the correct model?

**Comment by bartmch** [comment]
FYI:[test.txt](https://github.com/arpanmangal/CovidAID/files/4757905/test.txt)

**Comment by bartmch** [comment]
> Which pre-trained model did you use? Did you use the correct model?

I followed your steps. The only difference is that I had to make the model torch>1.0 compatible but I didn't change any weights or so, only the names of the layers - that shouldn't really matter right?
FYI: [test.txt](https://github.com/arpanmangal/CovidAID/files/4757922/test.txt)

**Comment by bartmch** [comment]
Apologies - I just figured out I was pointing to the wrong trained network! Everything looks good now.
![image](https://user-images.githubusercontent.com/10415569/84259303-579ae800-ab29-11ea-8939-f5234d1534d4.png)
